CONTRIBUTING.md

```diff
@@ -0,0 +1,187 @@
+# Contributing to react-crisp
+
+First off, thanks for taking the time to contribute!
+
+The following as a set of guidelines for contributing to react-crisp.
+These are mostly guidelines, not rules. Use your best judgment, and fell free
+to propose changes to this document in a pull request.
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the
+[react-crisp Code of Conduct](CODE_OF_CONDUCT.md). By participating, you
+are expected to uphold this code. Please report unacceptable behavior to
+[jd@mergify.io](mailto:jd@mergify.io).
```

Comment:
Might be better to use a non-personal email.
I've created `opensource@mergify.io`